packages/alfa-dom/test/contains.spec.tsx

```diff
@@ -0,0 +1,23 @@
+import { jsx } from "@siteimprove/alfa-jsx";
+import { test } from "@siteimprove/alfa-test";
+import { contains } from "../src/contains";
+
+test("Contains returns true when a node contains the specifed node using string query", t => {
+  const strong = <strong />;
+  const div = <div>{strong}</div>;
+
+  t.equal(contains(div, <body>{div}</body>, "strong"), true);
+});
+
+test("Contains returns true when a node contains the specifed node using object query", t => {
```

Comment:
The titles in this test file is a bit confusing - especially the last part of the titles. Alternatives could be:
`Contains returns true when element is contained in parent element.`
`Contains returns true when tag name is contained in parent element.`